### PR TITLE
Add conda to the build container

### PR DESCRIPTION
**Pull request by Tabrizian**

### CODE REVIEW

**Review by szalpal — APPROVED** [review]
Tested it on building against https://github.com/triton-inference-server/dali_backend/pull/133
Works.

**Review by szalpal — APPROVED** [review]

**Review by dyastremsky — DISMISSED** [review]

**Review by GuanLuo — DISMISSED** [review]
minoconda is only required during build stage?

**Review by tanmayv25 — DISMISSED** [review]

File: build.py [hunk 1]
```diff
@@ -838,6 +840,30 @@ def install_dcgm_libraries(dcgm_version, target_machine):
...
+def install_miniconda(conda_version, target_machine):
+    if conda_version == '':
+        fail(
+            'unable to determine default repo-tag, CONDA version not known for {}'
+            .format(FLAGS.version))
+    miniconda_url = f"https://repo.anaconda.com/miniconda/Miniconda3-{conda_version}-Linux-{target_machine}.sh"
+    if target_machine == 'x86_64':
+        sha_sum = "3190da6626f86eee8abf1b2fd7a5af492994eb2667357ee4243975cdbb175d7a"
+    else:
+        sha_sum = "47affd9577889f80197aadbdf1198b04a41528421aaf0ec1f28b04a50b9f3ab8"
```
Comment (rmccorm4) [inline build.py:852]: Just for my own curiosity, will these SHA sums have to be updated whenever the conda_version is bumped?

Comment (Tabrizian) [reply build.py]: > have to be updated whenever the conda_version is bumped?
Yes. I added a comment.

**Review by rmccorm4 — APPROVED** [review]

**Pull request merged**